Content.Server/Teleportation/LinkEntitiesCommand.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.Teleportation.Systems;
+using Robust.Shared.Console;
+
+namespace Content.Server._Stalker.Teleportation;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class LinkEntitiesCommand : IConsoleCommand
```

Comment:
Use ToolsheCommand (ping moony if u got questions)

Reply:
it complicated and hard :despair: